app/components/common/userInput/index.tsx

```diff
@@ -9,6 +9,8 @@ interface UserInputProps {
   type?: string;
   value: string;
   setValue: (str: string) => void;
+  handleBlur: (e: React.FocusEvent<HTMLInputElement>) => void;
+  handleFocus: (e: React.FocusEvent<HTMLInputElement>) => void;
```

Comment:
are these utilized in all the places where `UserInput` is used? otherwise they can be optional